docs/customresourcestate-metrics.md

```diff
@@ -272,19 +267,55 @@ spec:
         target_kind: [spec, targetRef, kind]
         target_name: [spec, targetRef, name]
       metrics:
-        - name: "annotations"
-          help: "Kubernetes annotations converted to Prometheus labels."
+        - name: "vpa_containerrecommendations_target"
+          help: "VPA container recommendations for memory."
           each:
             type: Gauge
             gauge:
-              path: [metadata, annotations]
-# This will output the following metric:
-# HELP kube_customresource_autoscaling_annotations Kubernetes annotations converted to Prometheus labels.
-# TYPE kube_customresource_autoscaling_annotations gauge
-# kube_customresource_autoscaling_annotations{customresource_group="autoscaling.k8s.io", customresource_kind="VerticalPodAutoscaler", customresource_version="v1", namespace="default",target_api_version="autoscaling.k8s.io/v1",target_kind="Deployment",target_name="hamster",verticalpodautoscaler="hamster-vpa"} 123
+              path: [status, recommendation, containerRecommendations]
+              valueFrom: [target, memory]
+              labelsFromPath:
+                container: [containerName]
+          commonLabels:
+            resource: "memory"
+            unit: "byte"
+        - name: "vpa_containerrecommendations_target"
+          help: "VPA container recommendations for cpu."
+          each:
+            type: Gauge
+            gauge:
+              path: [status, recommendation, containerRecommendations]
+              valueFrom: [target, cpu]
+              labelsFromPath:
+                container: [containerName]
+          commonLabels:
+            resource: "cpu"
+            unit: "core"
+```
+
+The above configuration was tested on [this](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/examples/hamster.yaml) VPA configuration, outputting the following metrics:
+
+```
+# HELP kube_customresource_vpa_containerrecommendations_target VPA container recommendations for memory.
```

Comment:
As you can see here, this will generate two different help outputs for the same title. This should either be postfixed with _cpu / _memory (if you want two help outputs) or 
```suggestion
# HELP kube_customresource_vpa_containerrecommendations_target VPA container recommendations for CPU and memory.
```